File: Sekigae.java
import java.util.*;

public class Sekigae {
    public static void main(String[] args) {
        List<String> list = Arrays.asList("たけお", "きたけー", "ぐっさん", "おっくん");
        Collections.shuffle(list);
        System.out.println(list);
    }
}

File: sekigae.pl
use strict;
use List::Util;
 
my @name_list = ( "おっくん", "きたけー", "たけお", "ぐっさん" );
my @s_name_list = List::Util::shuffle @name_list;
 
print join(", ", @s_name_list);
